### ISSUE TITLE
Fails to start: cannot import name __version__

### ISSUE THREAD

**Issue by yurivict** [paradoxxxzero/butterfly#142]
```
Traceback (most recent call last):
  File "/usr/local/bin/butterfly.server.py", line 313, in <module>
    from butterfly import application
  File "/usr/local/lib/python2.7/site-packages/butterfly/__init__.py", line 80, in <module>
    import butterfly.routes  # noqa: F401
  File "/usr/local/lib/python2.7/site-packages/butterfly/routes.py", line 34, in <module>
    from butterfly.terminal import Terminal
  File "/usr/local/lib/python2.7/site-packages/butterfly/terminal.py", line 36, in <module>
    from butterfly import __version__, utils
ImportError: cannot import name __version__
```

Version 3.1.0

**Comment by paradoxxxzero** [comment]
Yeah sorry about that I released a patch shortly after (3.1.2) can you
confirm it fixed it?

**Comment by yurivict** [comment]
Yes, this problem is resolved the the patch.

Thanks!
